odoo/addons/base/models/res_currency.py

```diff
@@ -173,7 +181,10 @@ def amount_to_text(self, amount):
         self.ensure_one()
         def _num2words(number, lang):
             try:
-                return num2words(number, lang=lang).title()
+                if lang == 'bg' and num2bg is not None:
+                    return num2bg.cyrillic(number)
```

Comment:
Just to make sure, I got you correctly. So, You actually recommend doing something similar as `Num2Word_AR_Fixed` to_cardinal and this should call the cyrillic functionality since we took the code of num2cyrillic, Right?

Reply:
And with that we don't need to add dependencies

Reply:
@xmo-odoo Done!